tests/regression/56-witness/66-ghost-alloc-lock.t

```diff
@@ -16,136 +16,6 @@
-        updates:
-        - variable: alloc_m559918035_locked
-          value: "0"
-          format: c_expression
-      - location:
-          file_name: 66-ghost-alloc-lock.c
-          file_hash: $FILE_HASH
-          line: 14
-          column: 3
-          function: t_fun
-        updates:
-        - variable: alloc_m861095507_locked
-          value: "1"
-          format: c_expression
-      - location:
-          file_name: 66-ghost-alloc-lock.c
-          file_hash: $FILE_HASH
-          line: 17
-          column: 3
-          function: t_fun
-        updates:
-        - variable: alloc_m861095507_locked
-          value: "0"
-          format: c_expression
-      - location:
-          file_name: 66-ghost-alloc-lock.c
-          file_hash: $FILE_HASH
-          line: 28
-          column: 3
-          function: main
-        updates:
-        - variable: multithreaded
-          value: "1"
-          format: c_expression
-      - location:
-          file_name: 66-ghost-alloc-lock.c
-          file_hash: $FILE_HASH
-          line: 30
-          column: 3
-          function: main
-        updates:
-        - variable: alloc_m559918035_locked
-          value: "1"
-          format: c_expression
-      - location:
-          file_name: 66-ghost-alloc-lock.c
-          file_hash: $FILE_HASH
-          line: 32
-          column: 3
-          function: main
-        updates:
-        - variable: alloc_m559918035_locked
-          value: "0"
-          format: c_expression
-      - location:
-          file_name: 66-ghost-alloc-lock.c
-          file_hash: $FILE_HASH
-          line: 33
-          column: 3
-          function: main
-        updates:
-        - variable: alloc_m861095507_locked
-          value: "1"
-          format: c_expression
-      - location:
-          file_name: 66-ghost-alloc-lock.c
-          file_hash: $FILE_HASH
-          line: 35
-          column: 3
-          function: main
-        updates:
-        - variable: alloc_m861095507_locked
-          value: "0"
-          format: c_expression
-  - entry_type: flow_insensitive_invariant
-    flow_insensitive_invariant:
-      string: '! multithreaded || (alloc_m861095507_locked || g2 == 0)'
-      type: assertion
-      format: C
-  - entry_type: flow_insensitive_invariant
-    flow_insensitive_invariant:
-      string: '! multithreaded || (alloc_m559918035_locked || g1 == 0)'
-      type: assertion
-      format: C
-  - entry_type: flow_insensitive_invariant
-    flow_insensitive_invariant:
-      string: '! multithreaded || (0 <= g2 && g2 <= 1)'
-      type: assertion
-      format: C
-  - entry_type: flow_insensitive_invariant
-    flow_insensitive_invariant:
-      string: '! multithreaded || (0 <= g1 && g1 <= 1)'
-      type: assertion
-      format: C
+  $ (yamlWitnessStrip < witness.yml) > new-stripped.yml
+  $ ./compare-ghost-alloc.sh 66-ghost-alloc-stripped.yml new-stripped.yml
+  The files are the same after renaming the variables.
```

Comment:
This moves the diffing into the script rather than having dune do it. Also it hides the additional stripping being done which means that the additional stripping is assumed to be correct: if it was wrong, then it strips both incorrectly and we'd never see.

So I'd prefer a simpler solution: the additional script just does renamings in the produced (and YAML stripped) witness and that's outputted.
Then the final thing is clearly visible in the test and the diff should also be obvious: `alloc_*` variables are replaced with something deterministic.